src/legacy/server/logging/log_format_json.test.js

```diff
@@ -97,6 +99,79 @@ describe('KbnLoggerJsonFormat', () => {
       expect(message).toBe('memory: 0.0B uptime: 0:00:00 load: [1.00 1.00 2.00] delay: 0.000');
     });
 
+    describe('with metadata', () => {
+      it('logs an event with meta data', async () => {
+        const event = {
+          data: attachMetaData('message for event', {
+            prop1: 'value1',
+            prop2: 'value2',
+          }),
+          tags: ['tag1', 'tag2'],
+        };
+        const result = await createPromiseFromStreams([createListStream([event]), format]);
+        const { level, message, prop1, prop2, tags, } = JSON.parse(result);
```

Comment:
__nit:__ what about this here and in new tests below?
```ts
expect(JSON.parse(result)).toEqual({
  '@timestamp': expect.any(String),
  message: 'message for event',
  prop1: 'value1',
  prop2: 'value2',
  tags: ['tag1', 'tag2'],
});
```

Reply:
don't want to get false negatives if we change log message format (rename this `@timestamp` for example)

Reply:
I see, but at the same time something can go wrong and we may expose data in the `result` that we wouldn't want to expose and this test won't catch this, but it's minor - up to you.